app/notebook/server/CalcWebSocketService.scala

```diff
@@ -43,6 +45,9 @@ class CalcWebSocketService(
     var calculator: ActorRef = null
     var wss: List[WebSockWrapper] = Nil
 
+    protected val notebookStartTime = LocalTime.now()
+    private var lastCellExecutionTime: Option[LocalTime] = None
```

Comment:
okay, but what about `1 day`? So we can leave it running for a while per default.
Since it's configurable, we can lower it to 60 minutes per instance